src/AngularConstructorParametersTransformerv10.ts

```diff
@@ -0,0 +1,35 @@
+import { ConfigSet } from "ts-jest/dist/config/config-set";
+import { TransformationContext } from "typescript";
+// this seems to not be included in a module?
+const { constructorParametersDownlevelTransform } = require('@angular/compiler-cli/src/tooling');
+
+
+/**
+ * Transformer ID
+ * @internal
+ */
+export const name = 'angular-ctor-parameters';
```

Comment:
i don't see this use anywhere. Is it required ?

Reply:
This is just following the convention of the [ts-jest hoist transformer](https://github.com/kulshekhar/ts-jest/blob/master/src/transformers/hoist-jest.ts). If this is unused, I will remove it.

Reply:
oh those are for internal `ts-jest` unit test. So indeed they can be removed here :)

Reply:
Ok, I thought these might be used by `jest` for some reason. Will remove them.